dotcom-rendering/src/components/Discussion/CommentContainer.tsx

```diff
@@ -105,28 +106,24 @@ export const CommentContainer = ({
 	body,
 	setBody,
 	reportAbuse,
+	expandCommentReplies,
 }: Props) => {
+	const responses = comment.responses ?? [];
+	const totalResponseCount = comment.metaData?.responseCount ?? 0;
```

Comment:
`comment.responses` --> actual number of responses
`comment.metaData?.responseCount` --> number of responses that can be shown

Is this correct?